be1-go/cli/utility/mod.go

```diff
@@ -0,0 +1,64 @@
+package utility
+
+import (
+	"fmt"
+	"net/url"
+
+	be1_go "popstellar"
+	"popstellar/hub"
+	"popstellar/network/socket"
+	"sync"
+
+	"github.com/gorilla/websocket"
+	"golang.org/x/xerrors"
+)
+
+// ConnectToSocket establishes a connection to another server's witness
+// endpoint.
+func ConnectToSocket(otherHubType hub.HubType, address string, h hub.Hub, wg *sync.WaitGroup, done chan struct{}) error {
```

Comment:
Why did you delete this function from the witness package and added it to a utility one?

Reply:
Exactly

Reply:
Exactly